### DISCUSSION TITLE
How to convert a given conformer into a molecule?

### DISCUSSION THREAD

**Discussion by RMeli** [rdkit/rdkit#4520] [Q&A] [answered]
What is the best way to convert a given conformer (from a molecule containing multiple conformers) into a standalone molecule?

I am currently doing the following
```
cmol = Chem.MolFromMolBlock(Chem.MolToMolBlock(mol, confId=ID))
```
but I was wondering if there is a better/more concise way?

**Comment by ptosco** [comment, marked as answer]
@RMeli 
Make a quick copy and then add the given conformer from the original molecule:
```
cmol = Chem.Mol(mol, True)
cmol.AddConformer(mol.GetConformer(ID), assignId=True)
```

**Reply by RMeli** [reply]
Thanks @ptosco , this is what I was looking for!
